serde-generate/Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "serde-generate"
-version = "0.19.3"
+version = "0.19.2"
```

Comment:
Did you mean to drop the version?

Reply:
oops. good catch

Reply:
I suppose it passes CI because minor versions can be silently promoted.